docs/src/components/productJoyUI/HeroShowcase.tsx

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react';
+import JoyBox from '@mui/joy/Box';
+import JoyButton from '@mui/joy/Button';
+import JoyBreadcrumbs from '@mui/joy/Breadcrumbs';
+import JoyLink from '@mui/joy/Link';
+import JoySheet from '@mui/joy/Sheet';
+import JoyTypography from '@mui/joy/Typography';
+import HomeRoundedIcon from '@mui/icons-material/HomeRounded';
+import DownloadRoundedIcon from '@mui/icons-material/DownloadRounded';
+import ChevronRightRoundedIcon from '@mui/icons-material/ChevronRightRounded';
+import OrderDashboardApp from 'docs/data/joy/getting-started/templates/order-dashboard/App';
+import OrderList from 'docs/data/joy/getting-started/templates/order-dashboard/components/OrderList';
+import MuiLogo from 'docs/data/joy/getting-started/templates/order-dashboard/components/MuiLogo';
+import ColorSchemeToggle from 'docs/data/joy/getting-started/templates/order-dashboard/components/ColorSchemeToggle';
+
+export function Desktop() {
```

Comment:
I tried that, but it started to feel complex to manage both the desktop & mobile previews losing the transform effect. Like, if I'm hovering over the desktop preview, which is sitting on the back, it'd make sense to put it in front so I can better interact with it. But that's exactly the complexity I tried to avoid.

To tame that a bit, I reduced the skewness so it's less weird to interact. Still, it's mostly meant to be illustrative, where the interactivity is a nice surprise.